File: example/src/main/java/com/example/supportutils/MainActivity.java
package com.example.supportutils;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.LinearLayout;
import android.widget.TextView;

import com.hendraanggrian.support.utils.view.ViewGroups;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        LinearLayout parent = (LinearLayout) findViewById(R.id.parent);
        TextView textView = ViewGroups.addView(parent, TextView.class, ViewGroups.WRAP_CONTENT, ViewGroups.WRAP_CONTENT);
    }

    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
    }
}

File: example/src/com/example/kota/MainActivity.kt
package com.example.kota

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.preference.PreferenceFragmentCompat
import com.example.kota.utils.findAndSetOnClickListener
import kota.OpenTransit
import kota.replace
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)
        supportFragmentManager.replace(R.id.container, Content(), OpenTransit)
    }

    class Content : PreferenceFragmentCompat() {
        override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
            addPreferencesFromResource(R.xml.activity_main)
            findAndSetOnClickListener("dialogs") { startActivity(Intent(context, DialogsActivity::class.java)) }
            findAndSetOnClickListener("texts") { startActivity(Intent(context, TextsActivity::class.java)) }
        }
    }
}